public/includes/config.php

```diff
@@ -23,3 +23,6 @@
 define('SEND_EMAIL_CC',  $configs['send_email_cc']);
 define('SMTP_EMAIL',  $configs['smtp_email']);
 define('SMTP_PASSWORD',$configs['smtp_password']);
+
+// API key
+define('MAP_API_KEY', $configs['map_API_key']);
```

Comment:
Can you let this be in small caps please to follow the existing convention
```suggestion
define('MAP_API_KEY', $configs['map_api_key']);
```